h/cli/commands/devserver.py

```diff
@@ -80,7 +80,7 @@ def devserver(https, web, ws, worker, assets, beat):
         m.add_process('ws', 'gunicorn --name websocket --reload --paste conf/development-websocket.ini %s' % gunicorn_args)
 
     if worker:
-        m.add_process('worker', 'hypothesis --dev celery worker --autoreload -l INFO')
+        m.add_process('worker', 'hypothesis --dev celery worker -l INFO')
```

Comment:
It looks like we don't need this as we have other ways of reloading files automagically when running in dev mode, if I'm understanding what this feature was intended for. (I know it was deprecated but just being thorough about whether we need to replace it with something else or not and I think the answer is no we don't.)

Reply:
We have autoreloading of the webpage/API-serving code in h via gunicorn's `--reload` flag. I don't think we have any replacement mechanism for `--autoreload` in place here, and I'm [not sure it was actually working anyway](https://github.com/celery/celery/issues/1658). I don't know what commonly used alternatives are but I found one suggestion to use [watchdog](http://avilpage.com/2017/05/how-to-auto-reload-celery-workers-in-development.html).